monitoring/docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+version: '3.9'
+
+# Default logging for all containers.
+x-logging:
+  &default-logging
+  driver: json-file
+  options:
+    # Add the container name to the log file
+    tag: "{{.ImageName}}|{{.Name}}"
```

Comment:
idk why you'd need image name here but okay still interesting solution ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)

Reply:
Grouping by container type (image) can be also useful. And can be not. Whatever.